Dangerfile

```diff
@@ -15,4 +15,14 @@ end
 failure("Please add labels to this PR.") if github.pr_labels.empty?
 
 checkstyle_format.base_path = Dir.pwd
-checkstyle_format.report("app/build/reports/detekt/detekt.xml")
\ No newline at end of file
+checkstyle_format.report("app/build/reports/detekt/detekt.xml")
+
+Dir.glob("app/src/main/res/layout").select { |e| File.file? e }.each do |file|
+    puts file.filename
+
+    hasMaterialBottomView = File.readlines(file).grep(/com.google.android.material.bottomnavigation.BottomNavigationView/).any?
```

Comment:
Can `file` be nested? I think we just need the fully qualified path here.

Could change this to be like:
```suggestion
    hasMaterialBottomView = File.readlines("app/src/main/res/layout/#{file}").grep(/com.google.android.material.bottomnavigation.BottomNavigationView/).any?
```